projects/cloud/app/models/user.rb

```diff
@@ -11,18 +11,14 @@ class User < ApplicationRecord
     omniauth_providers: [:github, :gitlab]
 
   # Associations
-  has_one :account, as: :owner, inverse_of: :owner, dependent: :destroy
+  has_one :account, as: :owner, inverse_of: :owner, dependent: :destroy, required: true
```

Comment:
This will cause Active Record to raise in case we introduce a regression in the future.